### PR TITLE
Making the status quo of the project more clear

### PR DESCRIPTION
**Pull request by maxim-tschumak**
Update of the readme file. As mentioned by @LappleApple, we have to state the status and goals of the project more clear.

### CODE REVIEW

**Comment by LappleApple** [comment]
Hey @maxim-tschumak, can you merge the PR and then I'll look/edit? It's a lot easier for me to see things directly inside the regular README.md. Thanks (and good job getting around to this). :)

**Comment by LappleApple** [comment]
👍

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
@LappleApple you can use this link btw to render the version of README from this PR: https://github.com/zalando-incubator/zally/blob/afb4edf4d782d33d57ed5c5c992a10b37c5b6f2e/README.md

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 Minimalistic API linter of awesomeness, bringing order to a sea of APIs.
 
+With the standard configuration, Zally checks your APIs against the rules defined in [Zalando RESTful Guidelines](http://zalando.github.io/restful-api-guidelines/).
+You can configure rules and enable/disable them on the server side.
+Using `x-zally-ignore` extension in the API definition it's also possible to disable rules for a specific API.
+You can also implement your own rules in Kotlin (in this case you have to build the server).
```
Comment (mkulak) [inline README.md]: > implement your own rules in Kotlin
Not only in Kotlin. Any JVM language will work.

Comment (netme) [inline README.md]: But most probably we won't accept contributions in any JVM language back :-)

**Pull request merged**